packages/lib/src/utils/money.ts

```
@@ -0,0 +1,4 @@
+export const parsePrice = (priceString: string) => {
+  const parsedPrice = parseFloat(priceString);
+  return isNaN(parsedPrice) ? 0 : parsedPrice;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)